Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py

```diff
@@ -161,34 +324,40 @@ def main():
         PARSE AND VALIDATE INTEGRATION PARAMS
     """
     # get the service API url
-    base_url = demisto.params().get('url')
-    token = demisto.params().get('token')
-
+    params = {key: value for key, value in demisto.params().items() if value is not None}
```

Comment:
What do you need it for?
You handle it within the Client

Reply:
passing all the parameters at once via this

Reply:
will retain this